gemrb/GUIScripts/pst/GUIREC.py

```diff
@@ -981,9 +992,8 @@ def OpenLevelUpWindow ():
 	Label = Window.GetControl (0x1000000E)
 	Label.SetText (str (GemRB.GetPlayerStat (pc, IE_LOCKPICKING)) + '%')
 	# Plus and Minus buttons
-	for i in range (8):
-		Button = Window.GetControl (16 + i)
-		Button.SetState (IE_GUI_BUTTON_LOCKED)
+	LUSkillsSelection.SetupSkillsWindow (pc, LUSkillsSelection.LUSKILLS_TYPE_LEVELUP, LevelUpWindow, RedrawSkills, [0,0,0], [1,1,1], 0, False)
```

Comment:
Is this fake level difference ok?

Reply:
No, you're absolutely right, I must have overlooked that. I will fix it.

Reply:
This actually ties directly into the task of using more of the common level up code where appropriate, so while this has been very helpful to get the skills routine tidied up, it might have to sit here temporarily while I work with that. I'd prefer to concentrate on the ultimate goal rather than start duplicating stuff just to get things working.

Reply:
ok, so let's postpone. Can I merge this now?

Reply:
It probably wont hurt, since the functionality wasn't actually there before, so I'll undraft it if you want to merge. I've been mostly circling around things needed for #176 , so it won't be staying like this very long.